demo/src/app/pages/demo/components/custom-column.component.ts

```diff
@@ -0,0 +1,60 @@
+import { Component, ViewChild, ElementRef } from '@angular/core';
+
+import { Cell, DefaultEditor, Editor } from '../../../../../../ng2-smart-table';
+
+@Component({
+  template: `
+    Name: <input [ngClass]="inputClass"
+            #name
+            class="form-control short-input"
+            [name]="cell.getId()"
+            [disabled]="!cell.isEditable()"
+            [placeholder]="cell.getTitle()"
+            (click)="onClick.emit($event)"
+            (keyup)="updateValue()"
+            (keydown.enter)="onEdited.emit($event)"
+            (keydown.esc)="onStopEditing.emit()">
+    Url: <input [ngClass]="inputClass"
+            #url
+            class="form-control short-input"
+            [name]="cell.getId()"
+            [disabled]="!cell.isEditable()"
+            [placeholder]="cell.getTitle()"
+            (click)="onClick.emit($event)"
+            (keyup)="updateValue()"
+            (keydown.enter)="onEdited.emit($event)"
+            (keydown.esc)="onStopEditing.emit()">
+    <div [hidden]="true" [innerHTML]="cell.getValue()" #htmlValue></div>
+    `
+})
+export class CustomColumnComponent extends DefaultEditor implements Editor{
```

Comment:
- do not have to write `implements Editor`. `DefaultEditor` is already implementing this interface.